--- app_unpacked/src/main-process/start.js	2016-11-26 15:33:42.639399618 +0900
+++ app_modified/src/main-process/start.js	2016-11-26 15:39:05.199403191 +0900
@@ -25,6 +25,15 @@
   const args = parseCommandLine(process.argv.slice(1))
   setupAtomHome(args)
   setupCompileCache()
+  
+  try {
+    var switches = require(path.join(process.env.ATOM_HOME, 'switches.json'));
+    switches.forEach(function (sw) {
+      if (sw.length > 0) {
+        app.commandLine.appendSwitch.apply(app.commandLine, sw);
+      }
+    });
+  } catch(e){}
 
   if (handleStartupEventWithSquirrel()) {
     return
