diff -urN app/src/main-process/atom-window.js app-modified/src/main-process/atom-window.js
--- app/src/main-process/atom-window.js	2017-09-07 10:31:04.000000000 +0900
+++ app-modified/src/main-process/atom-window.js	2017-09-07 10:42:14.000000000 +0900
@@ -58,6 +58,9 @@
       options = {
         show: false,
         title: 'Atom',
+        backgroundColor: "#00000000",
+        transparent: true,
+        frame: false,
         webPreferences: {
           backgroundThrottling: !this.isSpec,
           disableBlinkFeatures: 'Auxclick'
